tower/Cargo.toml

```
@@ -48,9 +48,9 @@ hedge = ["util", "filter", "futures-util", "hdrhistogram", "tokio/time", "tracin
 limit = ["tokio/time", "tokio/sync", "tokio-util", "tracing", "pin-project-lite"]
 load = ["tokio/time", "tracing", "pin-project-lite"]
 load-shed = ["pin-project-lite"]
-make = ["pin-project-lite", "tokio/io-std"]
+make = ["pin-project-lite", "tokio/sync"]
 ready-cache = ["futures-core", "futures-util", "indexmap", "tokio/sync", "tracing", "pin-project-lite"]
-reconnect = ["make", "tokio/io-std", "tracing"]
+reconnect = ["make", "tokio/sync", "tracing"]
```

Same here, `reconnect` doesn't seem to use `tokio` so this seems like it can be removed entirely.

```suggestion
reconnect = ["make", "tracing"]
```